examples/custom_client.py

```diff
@@ -0,0 +1,16 @@
+import os
+from indico import IndicoClient, IndicoConfig
+
+# Use environment variables
+os.environ["INDICO_PROTOCOL"] = "http"
+os.environ["INDICO_HOST"] = "foo.bar.com"
+
+# Use IndicoConfig
+my_config = IndicoConfig(
+    host="indico.my-company.com",  # Overrides environment variable
+    api_token_path="../path/to/custom_api_token.txt",
+    verify_ssl=False,
```

Comment:
that part shouldn't be necessary for most use cases we shouldn't show it ;p it's in the further docs

Reply:
sure, will remove